### PR TITLE
Add reference to additional nerves libraries

### PR DESCRIPTION
**Pull request by joelbyler**
## Title
Add reference to additional nerves libraries

## Description
Hoping to help folks find libraries they might need for use with nerves projects

## Commit message
Adding a link to the nerves libraries list on the nerves-project.org website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -560,6 +560,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
 
 * [bake](https://github.com/bakeware/bake) - Configure, compile and share systems, toolchains and linux firmware.
```
Comment (GregMefford) [inline README.md]: Hm... we should probably remove `bakeware` while you're in here. 😬 
That hasn't been relevant for quite some time.

File: README.md [hunk 2]
```diff
@@ -560,6 +560,7 @@ There are [other sites with curated lists of elixir packages](#other-awesome-lis
...
 * [nerves](http://nerves-project.org) - A framework for writing embedded software in Elixir.
+  * Additional nerves specific libraries may be found [here](https://nerves-project.org/libraries/)
```
Comment (GregMefford) [inline README.md]:
```suggestion
  * Additional Nerves-specific libraries may be found [here](https://nerves-project.org/libraries/)
```

File: README.md (hunk 1 above)
Comment (fhunleth) [reply README.md]: https://github.com/h4cc/awesome-elixir/pull/4523

Comment (joelbyler) [inline README.md]: @GregMefford @fhunleth what if we just swap out backware for @entone's firmata lib like so?

```suggestion
* [firmata](https://github.com/entone/firmata) - Firmata protocol implemented in Elixir
```

Comment (GregMefford) [reply README.md]: LGTM. 👍

**Pull request closed without merging**